src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,67 @@ redirects:
+
+## **Detect** problems in your system [#detect]
+
+Use the detect features to define the conditions in your system you want to be notified about, as well as to configure how anomalies are detected.
+
+* **[Alert conditions (policies)](/docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/create-edit-or-find-alert-policy/)**: Create new policies. Also, lists the policies where your conditions are created and organized. 
+* **[Anomaly detection](/docs/alerts-applied-intelligence/applied-intelligence/proactive-detection/expanded-anomaly-detection/)**: Manage your anomaly detection configurations. An anomaly is a behavioral trend that doesn't match the historical data for your system. You can add or edit the configurations here.
+
+## **Correlate** the data coming into your system [#correlate]
+
+Use the correlate features to define your data sources and to review and configure the logic that groups your incidents.
+
+* **[Sources](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources)**: Manage the data input sources you've chosen to analyze and be notified about. You can add new sources or configure existing ones.
+* **[Decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/)** Review your correlated incidents. You can also review, edit, and add decisions.
+
+## **Enrich and notify** your notifications with useful metadata [#enrich]
+
+Use the enrich and notify features to add additional metadata to your notifications, schedule when you don't want to be notified, and configure where your notifications get sent.
+
+* **[Muting rules](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/muting-rules-suppress-notifications/)**: Temporarily stop your alerts notifications.
+* **[Workflows](/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/incident-workflows/)**: Add more data to your notifications.
+* **[Destinations](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#2-configure-destinations)**: Configure where your applied intelligence notifications get sent.
+
+## Use **Settings** to review and manage your usage [#settings]
+
+Use the general settings page to review and manage your current incident intelligence usage.
+
+## **Alerts (classic)** [#alerts]
+
+Alerts (classic) lets you manage events, incidents, and notification channels with the UI you're used to using.
+
+* **[Events and incidents](/docs/alerts-applied-intelligence/new-relic-alerts/alert-incidents/view-violation-event-details-incidents/)**: Review alerts violations and other events.
+* **[Channels](/docs/alerts-applied-intelligence/new-relic-alerts/alert-notifications/notification-channels-control-where-send-alerts/)** Review your alerts notification channels and to create new channels.
```

Comment:
@paperclypse, looks like you need a colon following "Channels."

Reply:
Thanks for spotting those! Apparently I'm blinded by the **s :)